gobblin-kafka/src/main/java/gobblin/kafka/tool/SimpleKafkaConsumer.java

```diff
@@ -69,10 +65,17 @@ public SimpleKafkaConsumer(Properties props, KafkaCheckpoint checkpoint)
 
     schemaRegistry = KafkaSchemaRegistryFactory.getSchemaRegistry(props);
     deserializer = new LiAvroDeserializer(schemaRegistry);
+    /** TODO: Make Confluent schema registry integration configurable
+     * HashMap<String, String> avroSerDeConfig = new HashMap<>();
+     * avroSerDeConfig.put("schema.registry.url", "http://localhost:8081");
+     * deserializer = new io.confluent.kafka.serializers.KafkaAvroDeserializer();
+     * deserializer.configure(avroSerDeConfig, false);
+     *
+     **/
 
     Properties consumeProps = new Properties();
     consumeProps.put("zookeeper.connect", zkConnect);
-    consumeProps.put("group.id", "testConsumer2");
+    consumeProps.put("group.id", "gobblin-tool-" + System.nanoTime());
```

Comment:
Much harder to restart consumption. You have to delete offsets separately. 
This is much better if you are debugging issues with pulling from kafka because you don't have to worry about your checkpoint getting committed for you automatically. 

kafka 0.9 onwards has better consumers, but I wanted to be able to run this directly from gobblin, the ide etc.